libs/cde-visualization/src/lib/components/histogram/histogram.component.scss

```diff
@@ -35,3 +36,9 @@
     flex-grow: 1;
   }
 }
+
+::ng-deep {
+  .histogram-panel {
+    max-width: 27.5rem;
```

Comment:
Why?

Reply:
The other tooltips have this to limit the width of the dialog, should I take it out?